src/components/Tabs/Annotation/AnnotationFileInfo.tsx

```diff
@@ -51,12 +51,12 @@ export default function AnnotationFileInfo(props: AnnotationFileInfoProps): Reac
       const warningText = formatQuantityString(unparseableRows, "object", "objects");
       conversionWarnings.push(`- ${warningText} had non-numeric values in a metadata column and could not be parsed.`);
     }
-    const maxMismatchedData = Math.max(mismatchedTimes, mismatchedTracks);
+    const maxMismatchedData = Math.max(mismatchedTimes, mismatchedTracks, mismatchedLabels);
     if (maxMismatchedData >= 1) {
       const warningText = formatQuantityString(
         maxMismatchedData,
-        "object had a time or track that does",
-        "objects had times or tracks that do"
+        "object had a time, track, or label that does",
+        "objects had times, tracks, or labels that do"
```

Comment:
![image](https://github.com/user-attachments/assets/1e0be3e5-3587-4d02-b52a-77fb5925c2f5)

Appears in import view